packages/core/src/views/account-center/Micro.svelte

```diff
@@ -0,0 +1,100 @@
+<script lang="ts">
+  import { fade } from 'svelte/transition'
+  import { internalState$, wallets$ } from '../../streams'
+  import {
+    getDefaultChainStyles,
+    shortenAddress,
```

Comment:
There seems to be a handful of unused import, ` fade, shortenAddress, shortenEns, unrecognizedChainStyle`